.changeset/olive-mugs-wait.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': minor
+---
+
+Deprecated the Checkbox's `children` prop. Use the `label` prop instead. This matches the RadioButton component.
```

Comment:
Nit, current wording is also fine. Just thought we could mention that `label` has been added. But maybe that makes the whole thing less clear.

```suggestion
Added an optional `label` prop to the `Checkbox` component, and deprecated the legacy `children` prop. The `label` prop will become required in v7. This aligns with the `RadioButton` component's API.
```